### PR TITLE
Show bubble over nav icon for unread messages

### PR DESCRIPTION
**Pull request by agu3010**
@RocketChat/iOS
@rafaelks 
Closes #399 #287 

hackfix - Can be done in a better way, i.e. bubble for unacknowledged notifications

### CODE REVIEW

File: Rocket.Chat/Controllers/Subscriptions/SubscriptionsViewController.swift [hunk 1]
```diff
@@ -268,7 +271,7 @@ extension SubscriptionsViewController {
             if channelGroup.count > 0 {
                 groupInfomation?.append([
                     "name": String(format: "%@ (%d)", localized("subscriptions.channels"), channelGroup.count)
-                ])
+                    ])
```
Comment (rafaelks) [inline Rocket.Chat/Controllers/Subscriptions/SubscriptionsViewController.swift]: No needed change here.

File: Rocket.Chat/Controllers/Subscriptions/SubscriptionsViewController.swift [hunk 2]
```diff
@@ -250,16 +251,18 @@ extension SubscriptionsViewController {
...
             if favoriteGroup.count > 0 {
                 groupInfomation?.append([
                     "icon": "Star",
                     "name": String(format: "%@ (%d)", localized("subscriptions.favorites"), favoriteGroup.count)
-                ])
-
+                    ])
                 favoriteGroup = favoriteGroup.sorted {
-                    return $0.type.rawValue < $1.type.rawValue
+                return $0.type.rawValue < $1.type.rawValue
```
Comment (rafaelks) [inline Rocket.Chat/Controllers/Subscriptions/SubscriptionsViewController.swift]: No needed change here.

File: Rocket.Chat/Controllers/Subscriptions/SubscriptionsViewController.swift [hunk 3]
```diff
@@ -250,16 +251,18 @@ extension SubscriptionsViewController {
...
+                NotificationCenter.default.post(name: Notification.Name("All_Messages_Read_Notification"), object: nil)
             }
 
             if favoriteGroup.count > 0 {
                 groupInfomation?.append([
                     "icon": "Star",
                     "name": String(format: "%@ (%d)", localized("subscriptions.favorites"), favoriteGroup.count)
-                ])
-
+                    ])
```
Comment (rafaelks) [inline Rocket.Chat/Controllers/Subscriptions/SubscriptionsViewController.swift]: No needed change here.

File: Rocket.Chat/Controllers/Subscriptions/SubscriptionsViewController.swift [hunk 4]
```diff
@@ -250,16 +251,18 @@ extension SubscriptionsViewController {
                 }
 
                 groupSubscriptions?.append(unreadGroup)
+                NotificationCenter.default.post(name: Notification.Name("Unread_Message_Notification"), object: nil)
+            } else {
+                NotificationCenter.default.post(name: Notification.Name("All_Messages_Read_Notification"), object: nil)
```
Comment (rafaelks) [inline Rocket.Chat/Controllers/Subscriptions/SubscriptionsViewController.swift]: Can you use some `constant` for the notification name? Since this is being used in more than one place.

Comment (agu3010) [inline Rocket.Chat/Controllers/Subscriptions/SubscriptionsViewController.swift]: Ok sure.

File: Rocket.Chat/Controllers/Subscriptions/SubscriptionsViewController.swift (hunk 3 above)
Comment (agu3010) [inline Rocket.Chat/Controllers/Subscriptions/SubscriptionsViewController.swift]: The spacing was automatically done by Xcode.

Comment (rafaelks) [inline Rocket.Chat/Controllers/Subscriptions/SubscriptionsViewController.swift]: OK, but you don't need to commit it. Please, remove the unnecessary changes.

**Comment by rafaelks** [comment]
Closing the PR due to inactivity.
